$OpenBSD: patch-src_ml_gtkstock_c,v 1.1 2013/02/01 10:22:01 dcoppa Exp $

Add missing handling of new modifier keys (Meta, Super, Hyper)
(upstream git commit 579347d69a83703472ee8a46e522f5bb7b739d6b)

--- src/ml_gtkstock.c.orig	Thu Aug 23 12:37:48 2012
+++ src/ml_gtkstock.c	Fri Feb  1 09:46:59 2013
@@ -139,6 +139,8 @@ CAMLprim value ml_gtk_stock_lookup(value id)
   TESTANDCONS(MOD4); TESTANDCONS(MOD5);
   TESTANDCONS(BUTTON1); TESTANDCONS(BUTTON2); TESTANDCONS(BUTTON3);  
   TESTANDCONS(BUTTON4); TESTANDCONS(BUTTON5);
+  TESTANDCONS(SUPER); TESTANDCONS(HYPER);
+  TESTANDCONS(META); TESTANDCONS(RELEASE);
   stock_result = alloc_tuple(4);
   Store_field(stock_result,0,Val_string(r.stock_id));
   Store_field(stock_result,1,Val_string(r.label));
